lib/taskana-core/src/test/resources/taskana.properties

```diff
@@ -3,6 +3,7 @@ taskana.roles.admin=admin | uid=admin,cn=users,OU=Test,O=TASKANA
 taskana.roles.businessadmin=businessadmin | cn=business-admins,cn=groups,OU=Test,O=TASKANA
 taskana.roles.monitor=monitor | cn=monitor-users,cn=groups,OU=Test,O=TASKANA
 taskana.roles.taskadmin=taskadmin
+taskana.roles.taskrouter=cn=ksc-users,cn=groups,OU=Test,O=TASKANA | user-taskrouter
```

Comment:
It is probably not a good idea to add all ksc-users to this role. This could lead to unexpected behaviour if someone is not aware of this. 
As the name is not checked against LDAP you could probably use any name.

Reply:
Ok, I will use `cn=routers,cn=groups,OU=Test,O=TASKANA` as it doesn't exist yet :)